pyqttoolkit/views/datetime_range_selector.py

```diff
@@ -60,6 +66,30 @@ def __init__(self, parent, paging=False, link_manager=None, reset_enabled=True,
             self._layout.addWidget(self._back_one, 0, 0, 2, 1)
             self._layout.addWidget(self._forward_one, 0, 2, 2, 1)
 
+        if set_to_limit_buttons:
+            self._set_to_start = QPushButton(self.tr('Set to Start'), self)
+            self._set_to_start.setSizePolicy(QSizePolicy.Fixed, QSizePolicy.Fixed)
+            self._set_to_start.setFixedWidth(65)
+            self._set_to_end = QPushButton(self.tr('Set to End'), self)
+            self._set_to_end.setSizePolicy(QSizePolicy.Fixed, QSizePolicy.Fixed)
+            self._set_to_end.setFixedWidth(65)
+
+            self._layout.addWidget(self._set_to_start, 0, 2)
+            self._layout.addWidget(self._set_to_end, 1, 2)
+
+            def _handle_set_to_limit_clicked(widget, prop):
+                def _():
+                    widget.reset()
+                    prop.__get__(self).emit(widget.value)
+                return _
+
+            self._set_to_start.clicked.connect(
+                _handle_set_to_limit_clicked(
```

Comment:
I think this will result in memory leaks - we likely need to store a weakref to `self` inside `_handle_set_to_limit_clicked`